SPECS/apr/apr.spec

```diff
@@ -54,41 +56,57 @@ make -j1 check
 %exclude %{_libdir}/pkgconfig
 %{_bindir}/*
 
-%files  devel
+%files devel
 %defattr(-,root,root)
 %{_includedir}/*
 %{_libdir}/*.a
 %{_libdir}/*.so
 %{_libdir}/pkgconfig
 
 %changelog
+* Tue Feb 14 2022 Suresh Thelkar <sthelkar@microsoft.com> - 1.7.2-1
```

Comment:
Date is wrong.  It's 2023 now.

Reply:
Thanks @jslobodzian. I have addressed it. Please look at the latest commit.